src/App.js

```diff
@@ -90,9 +96,21 @@ export default function App() {
                             ),
                         };
                         episodes.push(parsedEpisode);
+                        if (parsedEpisode.category === "Huge BBC") {
+                            BBCEpisodes.push(parsedEpisode);
+                        }
+                        if (
+                            parsedEpisode.category ===
+                                "The Last Of Us Play The Last Of Us" ||
```

Comment:
you might be able to simplify this by saying something like `parsedEpisode.category.contains("The Last Of Us")`